frontend/device/input.lua

```diff
@@ -341,6 +341,14 @@ function Input:handleKeyBoardEv(ev)
         return
     end
 
+    -- A Button to turn on/off the frontlight
+    if keycode == "FLSwitch" and ev.value == EVENT_VALUE_KEY_RELEASE then
+        local Device = require("device")
+        local powerd = Device:getPowerDevice()
+        powerd:detectedFrontlightSwitchToggle()
+        return "FLSwitch"
```

Comment:
why?. You don't handle it in UIManager. Should be `return` instead?

Reply:
OK.